docs/explanations/architecture/styles.md

```diff
@@ -80,7 +58,7 @@ To build an experience like the one described above a block author needs a few p
 
 In essence, these are the essential mechanics a block author needs to care about for their block to be able to be styled by the user. While this can be done completely manually, there's an API that automates this process for common style needs: block supports.
 
-#### Block Supports API
+### Block Supports API
```

Comment:
```suggestion
### Block supports API
```

Reply:
Is "Block Supports" a proper noun?

Reply:
I would consider the "Block Supports API" a proper noun, but "block supports" as not a proper noun, much like block styles.